src/Containers/RightSideBar/ModifierTools.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { StandardView, AlternativeView } from './Views';
+import { EditText } from './EditText';
+import { EditImage } from './EditImage';
+
+// MODIFIER PROPS
+export interface ModifierProps {
+    title: string;
+    isStandardChosen: boolean;
+    isAlternativeChosen: boolean;
+    isText: boolean;
+    isImage: boolean;
+}
+// END OF MODIFIER PROPS
+
+export const ModifierTools: React.FC<ModifierProps> = ({
+    title,
+    isStandardChosen,
+    isAlternativeChosen,
+    isText,
+    isImage,
+}): React.ReactElement => {
+    if (isStandardChosen) {
+        if (isStandardChosen && isText) {
```

Comment:
I'm pretty sure there is a better way to render this, but check with Ralph? Should be more focused on the use of props to do such conditional rendering I think